exporter_test.go

```diff
@@ -15,6 +15,7 @@ package main
 
 import (
 	"fmt"
+	"github.com/prometheus/client_golang/prometheus"
```

Comment:
Please keep the usual order of imports in go: first packages from stdlib, newline, then external packages (both lists must be sorted).